src/main/java/core/basesyntax/model/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.model;
+
+public class Ball {
+    private String color;
+    private int number;
```

Comment:
let's create constructor for these fields

Reply:
added that constructor, and changed getRandomBall() method